mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -194,10 +203,15 @@ class LinearRegression(override val uid: String)
     if (handlePersistence) instances.unpersist()
 
     // TODO: Converts to sparse format based on the storage, but may base on the scoring speed.
-    copyValues(new LinearRegressionModel(uid, weights.compressed, intercept))
+    val model = new LinearRegressionModel(uid, weights.compressed, intercept)
+    val trainingResults = new LinearRegressionTrainingResults(
+      model.transform(dataset).select(model.getPredictionCol, model.getLabelCol),
+      lossHistory.result())
```

Comment:
Okay, lossHistory will still be the one returned in the scaled space. It's not easy and meaningful to convert it into original space.